cachi2/core/package_managers/gomod.py

```diff
@@ -448,9 +502,16 @@ def _find_missing_gomod_files(source_path: RootedPath, subpaths: list[str]) -> l
     return invalid_gomod_files
 
 
-def _resolve_gomod(
-    app_dir: RootedPath, request: Request, tmp_dir: Path
-) -> tuple[ParsedModule, Iterable[ParsedModule], Iterable[ParsedPackage]]:
+class ResolvedGoModule(NamedTuple):
+    """Contains the data for a resolved Go module."""
```

Comment:
Maybe add a note to make it clear this is an input ~package~ module (specified by the user)? We already have several `*Module`s :grin:

Reply:
Or maybe just call it ResolveResult, as it is being called in the variable names?

Reply:
I originally called it ResolveResult, but then updated it to match the [npm code ](https://github.com/containerbuildsystem/cachi2/blob/3f8752c89f46f3fbfe454a3fea564b7aa90f73d5/cachi2/core/package_managers/npm.py#L50)

Added the note though